addons/bus/static/src/js/services/assets_watchdog_service.js

```diff
@@ -0,0 +1,94 @@
+/** @odoo-module **/
+
+import { serviceRegistry } from "@web/webclient/service_registry";
+
+export const assetsWatchdogService = {
+    async deploy(env) {
+        const _assets = {};
```

Comment:
no need for underscore before variables and methods. These were only there to signal the "private" modifier. Since we are in a closure, the variables are not accessible from outside anyway